src/ape_console/_cli.py

```diff
@@ -113,6 +113,7 @@ def console(project=None, verbose=None, extra_locals=None):
             faulthandler.enable()  # NOTE: In case we segfault
 
     namespace = {component: getattr(ape, component) for component in ape.__all__}
+    namespace["ape"] = ape
```

Comment:
Thoughts on this:

```python

class Ape:
  def __getattr__(self, item):
      return getattr(ape, item)

  def __repr__(self):
      return "<ape>"

namespace["ape"] = Ape()
```

This makes it look like this in the console:

```
(dev) ➜  ape git:(patch-1) ✗ ape console

In [1]: ape
Out[1]: <ape>
```

Reply:
This is nice but doing it like so breaks tab-discovery, which is crucial for new users:

<img width="804" alt="image" src="https://user-images.githubusercontent.com/1091900/186711092-c6bd5609-1a12-457c-87ea-40aae396338e.png">

Maybe there's a way to set the repr on the actual module?

Reply:
Actually yes, in case we really want to go down that path: https://stackoverflow.com/a/1727209/1110703

Reply:
Sounds like this is a bad idea. Let's skip it! Was a fun thought experiment though. But probably best to keep modules as modules